fleet/api/http/endpoint.go

```diff
@@ -386,73 +386,3 @@ func removeAgentEndpoint(svc fleet.Service) endpoint.Endpoint {
 		return removeRes{}, nil
 	}
 }
-
-func listAgentBackendsEndpoint(svc fleet.Service) endpoint.Endpoint {
```

Comment:
This method _is_ generic to all backends so needs to stay here.

Reply:
Done